src/main/java/service/write/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package service.write.impl;
+
+import dao.StorageDao;
+import dao.impl.StorageDaoImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import service.write.ReportService;
+import service.write.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String DIRECTORY_PATH = "src/main/resources/";
+    private static final String CANT_WRITE_MESSAGE = "Can't write to file: ";
+    private ReportService reportService = new ReportServiceImpl();
+    private StorageDao storageDao = new StorageDaoImpl();
+
+    @Override
+    public void writeToFile(String filename) {
+        File file = new File(DIRECTORY_PATH + filename);
+        String report = reportService.createReport(storageDao.getAll());
```

Comment:
Common mistakes: Make your services independent and call them in main() method.
Change writeToFile(String filename) to writeToFile(String filename, String report) and create String report in main class